lib/config.go

```diff
@@ -461,9 +470,175 @@ func InitConfig() error {
+			"max_desire_healthy_worker_pct":        gAppConfig.MaxDesiredHealthyWorkerPct,
+		},
+		"R-W-SPLIT": {
+			"readonly_children_pct":          gAppConfig.ReadonlyPct,
+			"cfg_from_tns_override_rw_split": gAppConfig.CfgFromTnsOverrideRWSplit,
+		},
+		"STATE-LOG": {
+			//"state_log_file", // default config in occ.def, present in hera.txt but not present in config.go
+			"state_log_interval": gAppConfig.StateLogInterval,
+			"state_log_prefix":   gAppConfig.StateLogPrefix,
+		},
+		"RAC": {
+			"management_table_prefix": gAppConfig.ManagementTablePrefix,
+			"rac_sql_interval":        gAppConfig.RacMaintReloadInterval,
+			"rac_restart_window":      gAppConfig.RacRestartWindow,
+		},
+		"NO-CATEGORY": {
+			"database_type":  gAppConfig.DatabaseType, //	Oracle = 0; MySQL=1; POSTGRES=2
+			"cfg_from_tns":   gAppConfig.CfgFromTns,
+			"log_level":      gOpsConfig.logLevel,
+			"high_load_pct":  gAppConfig.HighLoadPct,
+			"init_limit_pct": gAppConfig.InitLimitPct,
+			//"page_alert",                 // default config in occ.def, present in hera.txt but not present in config.go
+			//"socket_timeout",             // present in occ.def for 1 or 2 occs only
+			//"standby_children_pct",       // default config in occ.def, present in hera.txt but not present in config.go
+			//"enable_occ_caching_routing", // default config in occ.def, present in hera.txt but not present in config.go
+			//"bits_to_match",              // present in occ.def for few OCCs. Can't find in config.go
+			//"max_batch_col_size",         // present in occ.def for few OCCs. Can't find in config.go
+			//"max_fetch_block_size",       // default and custom config in occ.def, present in hera.txt but not present in config.go
+			//"max_out_bind_var_size",      // present in occ.def for 1 or 2 occs only
+			//"ping_interval",              // present in occ.def for few OCCs.
+			"num_standby_dbs": gAppConfig.NumStdbyDbs,
+		},
+	}
+
+	//TODO: for local testing only. Remove before final push
+	//dir, _ := os.Getwd()
+	//fmt.Println("pwd: ", dir)
+
+	//Set the file search path to the current working directory
+	//_ = os.Chdir(dir + "/lib")
+	//if err != nil {
+	//	fmt.Println("Error:", err)
+	//	return nil
+	//}
+
+	for feature, configs := range whiteListConfigs {
+		switch feature {
+		case "BACKLOG":
+			if gAppConfig.BacklogPct == 0 {
+				continue
+			}
+		case "SHARDING":
+			if gAppConfig.EnableSharding == false {
+				continue
+			}
+		case "TAF":
+			if gAppConfig.EnableTAF == false {
+				continue
+			}
+		case "R-W-SPLIT":
+			if gAppConfig.ReadonlyPct == 0 {
+				continue
+			}
+		}
+
+		evt := cal.NewCalEvent("OCC_CONFIG", fmt.Sprintf(feature), cal.TransOK, "")
+		for cfg, val := range configs {
+			s := fmt.Sprintf("%v", val)
+			//fmt.Println("interface{} to string: ", s)
+			evt.AddDataStr(cfg, s)
+		}
+		evt.Completed()
+
+		configsMarshal, _ := json.Marshal(configs)
+		configsMarshalStr := string(configsMarshal)
+
+		//TODO: remove below logs before final push
```

Comment:
we can validate using file-based CAL log. so we can remove this JSON marshal code and server logging